README.md

```diff
@@ -84,6 +84,27 @@ mbedConnectorApi.startLongPolling(function(error) {
 });
 ```
 
+### Setup callback notifications
+```javascript
+var MbedConnectorApi = require('mbed-connector-api');
+var mbedConnectorApi = new MbedConnectorApi({
+  accessKey: /* Access Key */
+});
+
+// Replace `webappPutRouteCallback` with your web app's PUT request handler
+webappPutRouteCallback('/webhook', function (req, res) {
```

Comment:
Maybe replace this with `http.createServer` call so the example is complete, now it looks weird. It has a custom function name, but then uses express API to handle the request...

Also, does `handleNotifications` do a JSON.parse when the body type is string? If not, then it should, as express does not do this by default.

F.e.:

```js
var http = require('http');

var server = http.createServer(function (req, res) {
    if (req.method === 'PUT' && req.url === '/webhook') {
        var body = [];
        req.on('data', function (chunk) {
            body.push(chunk);
        }).on('end', function() {
            body = Buffer.concat(body).toString('utf-8');
            try {
                body = JSON.parse(body);
            }
            catch (ex) {
                res.statusCode = 500;
                res.write('Invalid JSON data');
                res.end();
                return;
            }

            mbedConnectorApi.handleNotifications(body);

            res.write('OK');
            res.end();
        })
    }

    // ...
});
```

Or just do it the express way, but then also explicitly say that we're doing express there...